packages/block-editor/src/components/list-view/style.scss

```diff
@@ -123,6 +127,35 @@
 		border-radius: 0;
 	}
 
+	&.is-nesting td {
+		background: rgba(var(--wp-admin-theme-color--rgb), 0.5);
+	}
+
+	&.is-nesting td:first-child {
+		border-top-left-radius: $radius-block-ui;
+		border-bottom-left-radius: $radius-block-ui;
+	}
+
+	&.is-nesting td:last-child {
+		border-top-right-radius: $radius-block-ui;
+		border-bottom-right-radius: $radius-block-ui;
+	}
+
+	&.is-above {
+		transition: transform 0.2s;
+		transform: translateY(-36px) !important;
+	}
+
+	&.is-below {
+		transition: transform 0.2s;
+		transform: translateY(36px) !important;
+	}
+
+	&.is-dragging {
```

Comment:
Note to self: if this rule stays, add a comment explaining why it's needed (we're effectively hiding the real appearance of the dragged blocks, without `display: none` so that there's no flicker when beginning to drag)